### PR TITLE
Wu2018group trowk

### PR DESCRIPTION
**Pull request by trowk**
Summary 1 for "Group Normalization , Wu, He; 2018" paper

### CODE REVIEW

File: _summaries/wu2018group_1.md [hunk 1]
```diff
@@ -1,19 +1,107 @@
...
+size on accuracy were also tested, the authors finding that a fixed size of 32
+groups are a variable size of 16 channels per group perfomed the best for their
+models.
+
+The authors also experimented with object detection and segmentation on the
+**COCO** dataset. Object detection and segmentation tend to involve higher
+resolution images, meaning smaller batch sizes tend to be used for these tasks.
+With 1 image per GPU for 8 GPUs being used for this task, group normalization
+outperformed batch normalization across the board.
+
```
Comment (sritank) [inline _summaries/wu2018group_1.md:88]: I believe they did an additional experiment using GN to train Mask R-CNN from scratch. They got better result than training it with BN and no pre-training.

File: _summaries/wu2018group_1.md [hunk 2]
```diff
@@ -1,19 +1,107 @@
...
+
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* Group Normalization outperforms Batch Normalization for small batch sizes
+* Group Normalization standardizes small groups of channels for each image
+* Group Normalization sufferes performance degradation compared to Batch
```
Comment (sritank) [inline _summaries/wu2018group_1.md:106]: What do you mean here? where do they see performance degradation in GN?

**Review by sritank — COMMENTED** [review]
Seems like you've captured the major points in the paper. I think a point to clarify is how they apply the GN in the different experiments. I don't believe they apply it directly on the input data in all cases.

File: _summaries/wu2018group_1.md (hunk 2 above)
Comment (trowk) [reply _summaries/wu2018group_1.md:106]: Changed it to say that "Group Normalization performs worse than Batch Normalization for larger batch sizes."

File: _summaries/wu2018group_1.md [hunk 3]
```diff
@@ -1,19 +1,107 @@
...
+different normalization methods had on accuracy for a neural network model
+training on the **ImageNet** dataset.
+
+<img width="500px" src="wu2018group_1c.PNG"/>
+
+Batch normalization performed the best with a batch size of 32 images per GPU
+(8 GPUs were used) outperforming group normalization by 0.5% on the validation
+set. However, group normalization achieved a 10.6% lower error rate with a batch
+size of 2, showing its effectiveness on lower batch sizes, The effects of group
+size on accuracy were also tested, the authors finding that a fixed size of 32
```
[the remark this answers is not in the document]
Comment (trowk) [reply _summaries/wu2018group_1.md:79]: In lines 78 and 79, I was specifically referring to their experiments where they varied the fixed group size and variable group size on ImageNet (the **Group Division** subsection on page 6 of the article). I will make it more clear that I am referring to a different experiment where they varied the number of groups. But yes, they do not apply the group normalization until after the first convolution and I'll be sure to make that more clear.

File: _summaries/wu2018group_1.md (hunk 1 above)
Comment (trowk) [reply _summaries/wu2018group_1.md:92]: I somehow overlooked this experiment. Thanks for mentioning it!

**Comment by trowk** [comment]
Thanks for the review! Let me know if you have any last minute changes you would like me to make!

**Review by sritank — APPROVED** [review]
Looks fine to me now
